wiki/FAQ/en.md

```diff
@@ -1,731 +1,81 @@
-[osu!catch icon link]: /wiki/shared/Ctb.gif "osu!catch icon"
-[osu!mania icon link]: /wiki/shared/Mania.gif "osu!mania icon"
-[osu_hit circles image]: /wiki/shared/osu_hitcircles.jpg "osu! hit circles"
-[osu_slider image]: /wiki/shared/osu_slider.jpg "osu! Slider"
-[osu_spinner image]: /wiki/shared/osu_spinner.jpg "osu! Spinner"
-[Taiko playfield image]: /wiki/shared/Taiko_playfield.jpg "osu!taiko playfield"
-[Taiko hit circles image]: /wiki/shared/Taiko_hitcircles.jpg "osu!taiko hit circles"
-[Taiko drumroll image]: /wiki/shared/Taiko_drumroll.jpg "osu!taiko drumroll"
-[Taiko spinner image]: /wiki/shared/Taiko_spinner.jpg "osu!taiko spinner (denden)"
-[osu!catch Playfield]: /wiki/shared/Catch_Playfield_27.jpg "osu!catch playfield difference based on CS"
-[Fruits image]: /wiki/shared/Catch_fruits.jpg "osu!catch fruits"
-[Fruit trails image]: /wiki/shared/Catch_trails.jpg "osu!catch fruit trails"
-[Bananas image]: /wiki/shared/Catch_bananas.jpg "osu!catch bananas"
-[Hyperfruit image]: /wiki/shared/Catch_hyperfruits.jpg "osu!catch hyperfruits"
-[osu!mania playfield image]:  /wiki/shared/Mania_playfield.jpg "osu!mania playfield"
-[osu!mania notes image]: /wiki/shared/Mania_notes.jpg "osu!mania notes"
-[osu!mania holdnotes image]: /wiki/shared/Mania_holdnotes.jpg "osu!mania hold notes"
-[osu! Interface image]: /wiki/shared/Interface_osu.jpg "osu! Interface"
-[osu!taiko Interface image]: /wiki/shared/Interface_taiko.jpg "osu!taiko Interface"
-[osu!catch Interface image]: /wiki/shared/Interface_ctb.jpg "osu!catch Interface"
-[osu!mania Interface image]: /wiki/shared/Interface_mania.jpg "osu!mania Interface"
-[osu!mania key layout image]: /wiki/shared/Mania_key_layouts.jpg "osu!mania key layout"
-[Options quickicons image]: /wiki/shared/Options_quickicons.jpg "Options quickicons"
-[Options keyboard image]: /wiki/shared/Options_keyboard.jpg "Options Input icon, Keyboard section"
-[Beatmap_not_owned image]: /wiki/shared/Beatmap_not_owned.png "Example of an .osu file content"
-[Mode Interface image]: /wiki/shared/Interface_mode.png "Mode change interface"
-[Slider ticks image]: /wiki/shared/Sliderticks.png "Image showing a slider with slider ticks, with top part during Edit and bottom part during play"
-[osu_smoke image]: /wiki/shared/osu_smoke.jpg "Smoke Usage"
-[osu_smoke_set image]: /wiki/shared/osu_smoke_set.jpg "Smoke in key bindings"
-
-<!-- Title -->
 # FAQ
 
-## osu! Introduction
+Got any questions or doubts? If so, we are here to help you! Below we have prepared a set of answers for the **frequently asked questions**.
 
-### I've never played this "Ouendan" thing - what is it?
-_[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the _Nintendo DS_.
-The gameplay consists of only three elements: tapping circles on the touchscreen, dragging a ball across a fixed path and rotating a spinner very fast.
-All these elements are timed to (covers of) popular Japanese songs.
-It looks like this on the _DS_:-
+## osu!
```

Comment:
This section heading is quite vague. At first, I thought it was relating to the game mode (since the `What is Ouendan?` section was below this one), but it seems to be relating more to the client itself.
Should it be `## osu! Client`?